quinn-proto/src/packet.rs

```diff
@@ -19,13 +19,13 @@ use crate::{
 // to inspect the version and packet type (which depends on the version).
 // This information allows us to fully decode and decrypt the packet.
 #[derive(Debug)]
-pub(crate) struct PartialDecode {
+pub struct PartialDecode {
```

Comment:
Should make this `#[doc(hidden)]` then (same for the other things in this commit).